inception/inception-api-annotation-experimental/src/main/ts/client/messages/response/relation/UpdateRelationResponse.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Technische Universität Darmstadt under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The Technische Universität Darmstadt
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export class UpdateRelationResponse
+{
+    relationAddress: number;
+    newFeatures : string[];
```

Comment:
Mind that features can be pretty complex - e.g. a slot feature is an array which contains elements that are actually another annotation type with a role and a target feature etc. Trying to support all of this would be pretty... complicated. The question is if you really need to work with features at this time or if we still stick with the feature editor on the right side of the screen for the time being. If we do features, then it would be sensible to stick with primitive features for the moment (string, int, boolean, double).